snippets/pid-limits.adoc

```diff
@@ -15,9 +15,17 @@
 
 :_mod-docs-content-type: SNIPPET
 
+// The PID limits are only configurable through the ROSA CLI. Once they are configurable through OCM, the conditionalization can be removed.
+
 [IMPORTANT]
 ====
-As of the {product-title} versions 4.8.35, 4.9.26, 4.10.6, the {product-title} default per-pod pid limit is `4096`. If you want to enable this PID limit, you must upgrade your {product-title} clusters to these versions or later. {product-title} clusters with prior versions use a default PID limit of `1024`.
+As of {product-title} 4.11, the default per-pod PID limit is `4096`. If you want to enable this PID limit, you must upgrade your {product-title} clusters to this version or later. {product-title} clusters running on earlier versions use a default PID limit of `1024`.
```

Comment:
I'm not sure what the procedure is but I would checkin with px on this support statement.

Reply:
Thanks - PX confirmed that this should be fine.